charts/linkerd2/templates/namespace.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if (.installNamespace) -}}
```

Comment:
This will need to go inside `{{ with .Values }}` to work.

Reply:
That's true, really. I hurried.